src/profile/main/tag/Google AdWords Remarketing - Prod_QA-UID26.js

```diff
@@ -61,18 +61,28 @@ try {
                     e = u.map[d].split(",");
                     for (f = 0; f < e.length; f++) {
                         prefix = /^ecomm\.|^hotel\.|^edu\.|^flight\.|^hrental\.|^job\.|^local\.|^listing\.|^travel\.|^dynx\./.exec(e[f]);
+                        prefix_custom = "custom.";
                         if (prefix !== null){
                             prefix = prefix[0].slice(0, -1);
                             u.data.params[prefix] = u.data.params[prefix] || {};
                             u.data.params[prefix][e[f].substr(prefix.length + 1)] = b[d];
-                        } else if (e[f].indexOf("custom.") == 0) {
-                            u.data.google_custom_params[e[f].substr(7)] = b[d];
+                        } else if (e[f].indexOf(prefix_custom) == 0) {
+                            u.data.google_custom_params[e[f].substr(prefix_custom.length)] = b[d];
                         } else {
                             u.data[e[f]] = b[d];
                         }
                     }
                 }
             }
+            if (b["isFlightConfirmation"]) {
```

Comment:
also, b['bookingWindow'] being null or undefined would evaluate true for your bookingWindow statement